### PR TITLE
Support for MultiPaper and Geyser

### PR DESCRIPTION
**Pull request by ChromMob**

### CODE REVIEW

File: src/main/java/ac/grim/grimac/utils/floodgate/FloodgateUtil.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package ac.grim.grimac.utils.floodgate;
+
+import org.geysermc.floodgate.api.FloodgateApi;
+
+import java.util.UUID;
+
+public class FloodgateUtil {
+
```
Comment (NoJokeFNA) [inline src/main/java/ac/grim/grimac/utils/floodgate/FloodgateUtil.java:8]: Remove this empty line

File: src/main/java/ac/grim/grimac/utils/floodgate/FloodgateUtil.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            CHECKED_FOR_FLOODGATE = true;
+        }
+
+        if (FLOODGATE_PRESENT) {
+            return FloodgateApi.getInstance().isFloodgatePlayer(uuid);
+        } else {
+            return false;
+        }
+    }
+
```
Comment (NoJokeFNA) [inline src/main/java/ac/grim/grimac/utils/floodgate/FloodgateUtil.java:29]: Remove this empty line

**Review by NoJokeFNA — CHANGES_REQUESTED** [review]

File: src/main/java/ac/grim/grimac/utils/floodgate/FloodgateUtil.java (hunk 1 above)
Comment (PureGero) [reply src/main/java/ac/grim/grimac/utils/floodgate/FloodgateUtil.java:8]: This change is already merged btw (this PR needs a rebase)
https://github.com/MWHunter/Grim/blob/2.0/src/main/java/ac/grim/grimac/utils/floodgate/FloodgateUtil.java

File: src/main/java/ac/grim/grimac/utils/floodgate/FloodgateUtil.java (hunk 2 above)
Comment (PureGero) [reply src/main/java/ac/grim/grimac/utils/floodgate/FloodgateUtil.java:29]: This change is already merged btw (this PR needs a rebase)
https://github.com/MWHunter/Grim/blob/2.0/src/main/java/ac/grim/grimac/utils/floodgate/FloodgateUtil.java

**Pull request closed without merging**